packages/py-moose-lib/moose_lib/internal.py

```
@@ -8,7 +8,7 @@
 """
 
 from importlib import import_module
-from typing import Literal, Optional, List, Any, Dict, Union, TYPE_CHECKING
+from typing import Literal, Optional, List, Any, Dict, Union
```

Ruff (0.14.14)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)